Milestone 7.1/script.js

```diff
@@ -0,0 +1,380 @@
+
+            for (let i = 0; i < sorted.length; i++) {
+
+                console.log(i);
+
+                const HTML = `<span class="fs-4  pb-2    border-bottom  border-secondary  mt-5 lh-lg">The Fibonnaci Of <b>${res.results[i].number}</b> is <b>${res.results[i].result}</b>. Calculated at: ${(new Date(res.results[i].createdDate)).toString()} <br> </span>`
+
+                resultsContainer.insertAdjacentHTML("beforeend", HTML)
+            }
+
+
+
+            // const HTML = `<span id="mySpan"  class="fs-5-text border-bottom  border-secondary pb-3 mt-4">The Fibonnaci Of <b>${sorted[i].number}</b> is <b>${sorted[i].result}</b>. Calculated at: ${(new Date(sorted[i].createdDate)).toString()} <br> </span>`
+
+
+        })
+
+        .catch(err => console.log(err))
+
+
+
+}
+
+
+
+
+
+// Milestone 7 
+
+
+let checkBoxMode = false
+
+
+form.addEventListener('submit', (e) => {
+    e.preventDefault()
+
+    if (userInput.value === '') return
+
+    if (!userInput.value === 42) {
+        userInput.style.borderColor = 'black'
+        result.style.textDecoration = 'underline'
+        result.style.fontSize = '25px'
+        result.style.fontWeight = 'bold'
+
+    }
+
+
+    hideError()
+
+
+
+    const number = userInput.value
+
+    if (number > 50) {
+        showError()
+        result.innerHTML = ''
+        return
+    }
+
+    // BOX UNCHECKED
+    if (!checkBox.checked) {
+
+        result.style.fontWeight = 'bold'
+        hideError()
+        fibonacci(userInput.value)
+
+    }
+
+
+    // BOX CHECKED
+    else {
+
+        calcFibo()
+
+
+
+    }
+
+
+
+
+
+
+
+})
+
+
+
+
+
+// Default Local Fibonnaci Function 
+const fibonacci = n => {
+
+    
+    userInput.style.borderColor = '#CCCCCC'
+    let a = 0, b = 1, c = n;
+
+    for (let i = 2; i <= n; i++) {
+        c = a + b;
+        a = b;
+        b = c;
+    }
+
+    // console.log(c);
+    result.innerText = c
+
+
+
+};
+
+
+
+function clearList() {
+
+    resultsContainer.innerHTML = ' '
+
+
+
+
+}
+
+// Milestone 7.1
+
+
+const select = document.getElementById('select')
+
+select.addEventListener('change', sortResults)
+
+
+async function sortResults(e) {
+
+if(!checkBox.checked) return 
+
+    const selectValue = e.target.value
+    console.log(e.target.value);
+
+    const data = fetch(resultsURL)
+    const res = await (await data).json()
+    console.log('res', res);
+
+
+    
+    switch (selectValue) {
+
+        case 'num-acc':
+            clearList()
+            const lowToHigh = res.results.sort((a, b) => {
+                return a.number - b.number
+            })
+
+            for (let i = 0; i < lowToHigh.length; i++) {
+                const HTML = `<span class="fs-4  pb-2    border-bottom  border-secondary  mt-5 lh-lg">The Fibonnaci Of <b>${lowToHigh[i].number}</b> is <b>${lowToHigh[i].result}</b>. Calculated at: ${(new Date(lowToHigh[i].createdDate)).toString()} <br> </span>`
+
+                resultsContainer.insertAdjacentHTML("beforeend", HTML)
```

Comment:
This functionality of looping and appending the HTML is being repeated in all of your switch cases. Why not make it into a function and then simply call the function with a different array as a parameter?